spec/mongo_batch_spec.rb

```diff
@@ -85,6 +86,16 @@ class Post
       expect(ids).to eq(posts.map(&:id).reverse)
     end
 
+    it 'does not enforce an order if one is already applied' do
+      posts = 1.upto(10).map { |n| FactoryGirl.create(:post, index: n) }
+
+      posts_in_batches = described_class
+                         .in_batches(Post.desc(:index), batch_size: 5)
+                         .map(&:to_a)
+      puts posts_in_batches.inspect
+      expect(posts_in_batches).to eq([posts[5..9].reverse, posts[0..4].reverse])
```

Comment:
Thinking of some of the lessons in [Working Effectively with Unit Tests](https://leanpub.com/wewut), is there a way to assert on simple values instead?

```ruby
it 'does not enforce an order if one is already applied' do
  posts = (1..10).map { |n| FactoryGirl.create(:post, index: n) }
  indexes = described_class.in_batches(Post.desc(:index), batch_size: 5).map do |batch|
    batch.map(&:index)
  end

  expect(indexes).to eq([[10, 9, 8, 7, 6], [5, 4, 3, 2, 1]])
end
```